modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/fleshlight.dm

```diff
@@ -54,30 +59,33 @@
 	. = ..()
 	if(!istype(target))
 		return
+	if(target.stat == DEAD)
```

Comment:
Did- someone use toys on a dead body that necessitate this check?

Reply:
![F_PstTAWgAALkS9](https://github.com/user-attachments/assets/ef1556d8-8571-4263-b051-261d247560bd)